### DISCUSSION TITLE
Please explain how to configure AC Dimmer

### DISCUSSION THREAD

**Discussion by Portia-Lin** [arendst/Tasmota#16801] [Q&A]
Hello. I want to make an AC dimmer. But I don't understand how to configure the esp8266 and the instructions are quite confusing. 
According to the instructions, I should configure GPIO4 as "Сounter 1". GPIO12 - "PWM 1". Next, I should physically connect these two outputs of the ESP8266 module and connect them to the PWM input of the RobotDyn dimmer module. I have to configure GPIO13 as "counter 4" and connect it to the Zero Cross output of the dimmer module. Next, I have to enter the following commands in the console:
`SETOPTION99 1
PWMFREQUENCY 100`
I am attaching a photo of the connection:
https://i.im.ge/2022/10/12/2FPysx.IMG-20221012-164433.jpg
Is that all I have to do to get an AC dimmer?
And a few more questions. Can I use any GPIOs other than the ones listed in the manual? I also noticed in the output settings section the name of the output is "ZC Pulse" what is this setting for? Thank you for your attention

**Comment by radudorurus** [comment]
There is another discussion more detailed
https://github.com/arendst/Tasmota/discussions/14978
